common/src/test/java/org/apache/sedona/common/FunctionsTest.java

```diff
@@ -561,6 +561,27 @@ public void getGoogleS2CellIDsAllSameLevel() {
         assertEquals(expects, levels);
     }
 
+    @Test
+    public void testS2ToGeom() throws ParseException {
+        Geometry target = GEOMETRY_FACTORY.createPolygon(
```

Comment:
Can you clean up all these comments and println?

Reply:
My bad, I forgot about them.